feat(port): add zed

dustessavdh opened the pull request:
#### Description

This PR: Adds support for the editor [Zed](https://zed.dev/).

The installation steps are written as if the package is already published.
To actually publish the package we would have to follow the steps listed in the [extensions repo of Zed](https://github.com/zed-industries/extensions/blob/main/AUTHORING_EXTENSIONS.md#publishing-your-extension).
Since I don't want to make a PR without asking, I haven't done this yet.

For local installation, you can follow the following steps:
1. Open the **Command Palette** with <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> or <kbd>⌘</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>
2. Open the **Extensions** view with the `zed: extensions` command
3. Click on the `Install Dev Extension` button
4. Select the [packages/zed](https://github.com/dustessavdh/aura-theme/tree/feat/zed/packages/zed) folder
5. Open the **Command Palette** again
6. Open the **Theme selector** with the `theme selector: toggle` command
7. Search for `Aura` and choose an `Aura` variant.

#### Assets
Logo:
![image](https://github.com/daltonmenezes/aura-theme/assets/25427808/40502d96-5b40-413e-ab70-fe0310badf3b)

Screenshot:
![image](https://github.com/daltonmenezes/aura-theme/assets/25427808/5aa51ea1-6820-4816-8eb0-e0fe6f9b1ada)


#### Checklist

- [x] PR description included
- [x] I've read the [documents](https://github.com/daltonmenezes/aura-theme#documentation)
- [x] I've created or commented in an issue related to this port asking to work on it
- [x] I know I shouldn't change any files in the packages folder manually
- [x] I've added this port at the end of the related category list in the [main README](https://github.com/daltonmenezes/aura-theme/blob/main/README.md)
- [x] I've attached an image to the icon of the app that this port is related to
- [x] I've attached a screenshot with a good zoom in fullscreen showing this port in action

daltonmenezes commented:
Hi @dustessavdh thanks, you can publish it for sure!

Also, please update the icon and banner to:

![icon](https://github.com/daltonmenezes/aura-theme/assets/1149845/85b87c48-0a8a-4276-b14e-2262b4848884)
![banner](https://github.com/daltonmenezes/aura-theme/assets/1149845/efd4b26e-e172-471c-8b87-35f11bf43ada)

dustessavdh commented:
Hi @daltonmenezes, I've updated the corresponding files!
I've also been trying to add it to the extensions repo, but I haven't fully figured out how to include this repo since they use git submodules.
I can add this repo as a submodule and use spare-checkout to only include this package, but it will still be under `packages/zed`, and I think the files must be in the root of the submodule.

I've added a question under https://github.com/zed-industries/extensions/issues/36 to ask more about this

daltonmenezes commented:
Thank you so much @dustessavdh 💜
